### ISSUE TITLE
False positive -  `warning: Missing else branch.`

### ISSUE THREAD

**Issue by vemv** [redplanetlabs/rama-clj-kondo#11]
This `if` causes a `warning: Missing else branch.`:

```clj
(defmodule MyModule [setup topologies]
  (declare-depot setup *foo-depot partition/foos)
  (if TEST?
    (declare-depot setup *bar-depot :random)
    (declare-tick-depot setup *baz-depot (* 1000 60 60))))
```

Thanks - V
